spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/storage_manager/cloud_volume_spec.rb

```diff
@@ -0,0 +1,37 @@
+describe ManageIQ::Providers::IbmCloud::PowerVirtualServers::StorageManager::CloudVolume do
+  let(:ems) do
+    FactoryBot.create(:ems_ibm_cloud_power_virtual_servers_cloud, :provider_region => "us-south")
+  end
+  let(:StorageManager) { FactoryBot.create(:ems_IbmCloud_StorageManager, :parent_ems_id => ems_cloud.id) }
+  let(:cloud_volume) { FactoryBot.create(:cloud_volume_IbmCloud, :ext_management_system => StorageManager ) }
+
+  describe "cloud volume operations" do
+    context "#delete_volume" do
+      it "deletes the cloud volume" do
+        stubbed_responses = {
+          :ec2 => {
```

Comment:
`:ec2` is an Amazon reference and should be modified

Reply:
Hi jay in this ec2 place what i want to do modify, I am trying to Ibmcloud,powervc like that,
even i am getting same error so what i need to change here?